docs/manuals/run-deploy/deploy/azure.md

```diff
@@ -1,4 +1,83 @@
 # Deploy on Azure
 
-TODO: [taipy-doc#397](https://github.com/Avaiga/taipy-doc/issues/397)
+[Azure] (https://azure.microsoft.com/en-us), one of the world's leading providers of cloud services, is the perfect platform for deploying Taipy applications. It allows developers to stay focused on their application bringing effortless scalability, monitoring and security natively. The following documentation shows how to deploy on Azure App Service and get an application into the hands of the end-user.
```

Comment:
"the perfect platform" is maybe a little bit too strong?
"a very relevant choice"?